### PR TITLE
Liv add color

### PR DESCRIPTION
**Pull request by liviutudor**
Added support for :

* background color -- via `bgColor` in configuration options
* text color -- via `textColor` in configuration options.

### CODE REVIEW

**Comment by bostrom** [comment]
@liviutudor Hey! Thanks for the PR, nice addition! And good job on the tests, although there seems to be some issues with Travis still, you think you can fix it? Finally, if you could squash all commits into a single one and force push the feature as one commit? Thanks!! 👍

**Comment by liviutudor** [comment]
@bostrom Hey, yup, working on the fixes for the tests -- it's bizarre as it passes locally on my Mac but looking into it now.
I'll have a look to see how I can squash the commits though.

**Comment by liviutudor** [comment]
ok squashed commits and managed to fix the tests - phew :)

File: test/helpers/setup.js [hunk 1]
```diff
@@ -12,3 +12,32 @@ global.assert = chai.assert;
...
+  var s = intVal.toString(16);
+  if (s.length == 1) {
+    s = "0" + s;
+  }
+  return s;
+}
+
+global.extractColors = function(image) {
+  var pixels = image.frames[0].data;
+  var colorMap = new Map();
```
Comment (bostrom) [inline test/helpers/setup.js]: Map is ES6, and we're not there yet with this project. Could you change it to a plain object?

Comment (bostrom) [inline test/helpers/setup.js]: Or would it be possible to utilize some existing library for reading pixels, e.g. https://www.npmjs.com/package/get-pixels?

In any case, the code here is obscure enough to require some explanatory comments 😉

**Comment by bostrom** [comment]
@liviutudor Hey I added eslint to the project to enforce some coding styles. Apparently it conflicts with your PR, sorry about that.

File: lib/text-to-image.js [hunk 2]
```diff
@@ -16,14 +16,24 @@ var fs = require('fs'),
...
 
-  var textData = createTextData(content, conf.maxWidth - conf.margin, conf.fontSize, conf.lineHeight);
+  // select background or use default if not specified
+  var background = '#fff';
+  if (conf.bgColor) {
+    background = conf.bgColor;
+  }
+
+  var textColor = '#000';
+  if (conf.textColor) {
```
Comment (liviutudor) [inline lib/text-to-image.js]: yup sure, good catch!

**Comment by bostrom** [comment]
@liviutudor I found some time to check out and tweak your PR myself. Thanks for the awesome contribution, it's now merged!

**Comment by liviutudor** [comment]
@bostrom thanks for doing this I was busy this weekend and didn't get to it.

**Pull request merged**